src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -57,7 +57,8 @@ def mds_counters(self, fs_id):
             "mds.imported_inodes",
             "mds.inodes",
```

Comment:
Do we still need to send this information in the response?
Doesn't look like it is being used in the frontend anymore.

Reply:
Yes, for chart in fs detail tab. But only two of them are used.
e.g. https://tracker.ceph.com/issues/37579